src/data-access/apiBasedLangCodeRetriever.ts

```diff
@@ -4,11 +4,15 @@ export type WbGetClaimsResponse = {
 	claims: StatementMap;
 };
 
-function getValueFromClaim( claim: Statement ): string | false {
+function getStringFromClaim( claim: Statement ): string | false {
```

Comment:
Sorry, I didn’t notice this before – I would also avoid the term “claim” and just use “statement” throughout. (Same goes for the `( claim ) =>` filter functions below.)

Reply:
mh, right. I think I remember somthing about deprecating the term "claim" and replacing it with "statement". I'll change it where I can

Reply:
I think, I've replaced all the ones that I can